src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -62,13 +63,32 @@
     private final String apiUri;
     private String pattern = ".*";
 
+    private @CheckForNull String includes;
+    private @CheckForNull String excludes;
+
     @DataBoundConstructor public GitHubSCMNavigator(String apiUri, String repoOwner, String scanCredentialsId, String checkoutCredentialsId) {
         this.repoOwner = repoOwner;
         this.scanCredentialsId = Util.fixEmpty(scanCredentialsId);
         this.checkoutCredentialsId = checkoutCredentialsId;
         this.apiUri = Util.fixEmpty(apiUri);
     }
 
+    public @Nonnull String getIncludes() {
+        return includes != null ? includes : DescriptorImpl.defaultIncludes;
+    }
+
+    @DataBoundSetter public void setIncludes(@Nonnull String includes) {
+        this.includes = includes.equals(DescriptorImpl.defaultIncludes) ? null : includes;
+    }
+
+    public @Nonnull String getExcludes() {
```

Comment:
As example the number of different contributors https://github.com/jenkinsci/pipeline-plugin/graphs/contributors if you would exclude `.md` updates  you would become alone. For jenkins future having one single developer as SPOF sounds very bad. 
PS i would say nothing about your personal preference for plugins that doesn't claimed to be jenkins future.